docs/factory_boy.md

```diff
@@ -2,7 +2,7 @@
 
 https://factoryboy.readthedocs.io/en/stable/index.html
 
-As a fixtures replacement tool, it aims to replace static, hard to maintain fixtures with easy-to-use factories for 
+As a fixtures replacement tool, it aims to replace static, hard to maintain fixtures with easy-to-use factories for
```

Comment:
This file should be in this branch.

Reply:
My bad, I meant this file shouldn't be in this branch since it is about FactoryBoy and this branch is about image field.